Sources/Fuzzilli/Core/CodeTemplates.swift

```diff
@@ -0,0 +1,167 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// These CodeTemplates create more complex code samples and are better
+// suited for the HybridFuzzing mode.
+public let CodeTemplates: [CodeTemplate] = [
+    CodeTemplate("JIT1Function") { b in
+        let genSize = 3
+
+        // Generate random function signatures as our helpers
+        var functionSignatures = CodeTemplate.generateRandomFunctionSignatures(forFuzzer: b.fuzzer, n: 2)
+
+        // Generate random property types
+        CodeTemplate.generateRandomPropertyTypes(forBuilder: b)
```

Comment:
I think you should modify the PropertyAssignmentGenerator to take property types into account, i.e. something like
```
...
var propertyType = b.type(ofProperty: propertyName)
// TODO unify the .unknown => .anything conversion
if propertyType == .unknown {
    propertyType = .anything
}
let value = b.randVar(ofType: propertyType)
...
```